src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormDetails.tsx

```diff
@@ -325,13 +326,28 @@ export const MyCollectionArtworkFormDetails: React.FC = () => {
             title="City"
             name="artworkLocation"
             placeholder="City where artwork is located"
-            maxLength={256}
+            maxLength={500}
```

Comment:
**question:** Is this change intentional?

Reply:
The `provenance` input should be limited to 500 as well ([Figma design](https://www.figma.com/file/99wX65Prz2lTsz9xLm5IrU/Artwork-Detail-Page?node-id=614%3A14139&t=aKmdrtYs9KzSNViV-0)).

Reply:
Yes this was intentional, I believe it is specified in the Figma file